webservices/docs.py

```diff
@@ -469,3 +470,187 @@
+EMPLOYER = 'Employer of contributor as reported on the committee\'s filing'
+OCCUPATION = 'Occupation of contributor as reported on the committee\'s filing'
+CONTRIBUTOR_NAME = 'Name of contributor'
+CONTRIBUTOR_CITY = 'City of contributor'
+CONTRIBUTOR_STATE = 'State of contributor'
+CONTRIBUTOR_EMPLOYER = 'Employer of contributor, filers need to make an effort to gather this information'
+CONTRIBUTOR_OCCUPATION = 'Occupation of contributor, filers need to make an effort to gather this information'
+CONTRIBUTOR_ZIP = 'Zip code of contributor'
+IS_INDIVIDUAL = 'Restrict to non-earmarked individual contributions where memo code is true. \
+Filtering individuals is useful to make sure contributions are not double reported and in creating \
+breakdowns of the amount of money coming from individuals.'
+
+
+
+# schedule B
+RECIPIENT_NAME = 'Name of the entity receiving the disbursement'
+RECIPIENT_ID = 'The FEC identifier should be represented here if the entity receiving \
+the disbursement is registered with the FEC.'
+
+# communication cost and electioneering
+SUPPORT_OPPOSE_INDICATOR = 'Explains if the money was spent in order to support or oppose a candidate or candidates. (Coded S or O for support or oppose.) This indicator applies to independent expenditures and communication costs.'
+
+# schedule B
+PURPOSE = 'Purpose of the expenditure'
+
+# dates
+DUE_DATE = 'Date the report is due'
+CREATE_DATE = 'Date the record was created'
+UPDATE_DATE = 'Date the record was updated'
+ELECTION_DATE = 'Date of election'
+ELECTION_YEAR = 'Year of election'
+#? TODO: add more categories
+ELECTION_TYPE = 'Election type \n\
+Convention, Primary, General, Special, Runoff etc.'
+SENATE_CLASS = 'Senators run every six years and each state has two senators. General elections \
+are held every 2 years. The Senate elections are staggered so there are three classes of Senators \
+In a given state, only one Senate seat is up for election at a time and every six years, there is \
+not a senate election in a given state. Thus, the Senate is broken up in to three groups called \
+classes. Senators in the same class are up for election at the same time. Sometimes it is a bit \
+less straight forward when, because there are situations in which there is a special election to \
+fill a vacancy in the Senate. In those cases class refers to the seat groupings and not the time \
+of the election.'
+
+# filings
+DOCUMENT_TYPE = 'Type of form, report or documents'
+ENDING_IMAGE_NUMBER = 'Image number is an unique identifier for each page the electronic or paper \
+report. The last image number corresponds to the image number for the last page of the document.'
+IMAGE_NUMBER = 'An unique identifier for each page the electronic or paper \
+report.'
+
+
+# Reports and Totals that are straight forward
+
+def add_period(var):
+    return var + ' total for the reporting period'
+
+
+def add_ytd(var):
+    return var + ' total for the year to date'
+
+
+def make_generic_def(var):
```

Comment:
I'm not sure I see the point of this helper. It looks like we're more or less copying the column name into the documentation field, modulo some spacing cleanup. But we aren't actually adding any information to the documentation--we're showing the same information twice. Imagine this in the download manifest:

aggregate_amount_personal_contributions_general: Aggregate amount personal contributions general